tests/application/test_send_notification_usecase.py

```
@@ -0,0 +1,65 @@
+        self.send_is_called = True
+        self.send_is_called_count += 1
+        notification.mark_as_sent()
+
+
+class FakeTestingFactory:
+    def __init__(self, sender: NotificationSender) -> None:
+        self.target_type: str = "testing"
+        self.sender = sender
+
+    def make_notificaton(
+        self, title: str, content: str, target: str, priority: Literal[0, 1, 2, 3] = 0
```

Ruff (0.13.3)

Unused method argument: `target`

(ARG002)